src/ios-deploy/ios-deploy.m

```diff
@@ -531,7 +553,12 @@ CFStringRef copy_device_support_path(AMDeviceRef device, CFStringRef suffix) {
     if (CFStringCompare(CFSTR("AppleTV"), deviceClass, 0) == kCFCompareEqualTo) {
       deviceClassPath[0] = CFSTR("Platforms/AppleTVOS.platform/DeviceSupport");
       deviceClassPath[1] = CFSTR("tvOS DeviceSupport");
-    } else {
+    }
+    else if (CFStringCompare(CFSTR("Watch"), deviceClass, 0) == kCFCompareEqualTo) {
+      deviceClassPath[0] = CFSTR("Platforms/WatchOS.platform/DeviceSupport");
+      deviceClassPath[1] = CFSTR("watchOS DeviceSupport"); // ???
```

Comment:
Was there some confusion with this line?

Reply:
Could not locate the directory initially, so got confused if it exists. Did verify it now, so I'll remove the comment.

Reply:
Removed the comment